package/MDAnalysis/lib/distances.py

```diff
@@ -390,6 +390,131 @@ def self_distance_array(reference, box=None, result=None, backend="serial"):
     return distances
 
 
+def capped_distance(reference, configuration, max_cutoff, min_cutoff=None, box=None, method=None):
+    """Returns the list of indices of reference and configuration
+     atoms within a fixed distance and corresponding distances.
+
+    If a box is supplied, then a minimum image convention is used to evaluate
+    the distances.
+
+    An optional keyword for the method is provided to allow usage of other
+    implemented methods. Currently pkdtree and bruteforce
+    can be used. Depending on the number of particles
+    and distribution, the times may vary significantly.
```

Comment:
it's not clear what's meant by `the times may vary`.  Make it clear that there's some automatic guessing of the best method, but users can also override this if they think they know better